providers/implementations/ciphers/cipher_aes_hw.c

```diff
@@ -130,7 +130,7 @@ static const PROV_CIPHER_HW aes_##mode = {                                     \
     cipher_hw_aes_copyctx                                                      \
 };                                                                             \
 PROV_CIPHER_HW_declare(mode)                                                   \
-const PROV_CIPHER_HW *PROV_CIPHER_HW_aes_##mode(size_t keybits)                \
+const PROV_CIPHER_HW *PROV_CIPHER_HW_aes_##mode(ossl_unused size_t UNUSED_keybits)    \
```

Comment:
The change is breaking the S390 CI build.

Reply:
i've just tested locally with Linux and macOS. Although being a former IBMer, i have no S/390 at home. These macros are quite nested. Hope, this was the only of such cases, otherwise we should reconsider these cases.

Reply:
Not many folks have access to an S/390 :)  I expect that other S/390 AES functions will be similar.

Reply:
the fix covers all AES *CIPHER_HW_* cases, not only S/390. Is the coverage of the CI builds sufficiently complete?